ci/scripts/r_docker_configure.sh

```diff
@@ -91,8 +91,9 @@ if [ -f "${ARROW_SOURCE_HOME}/ci/scripts/r_install_system_dependencies.sh" ]; th
   "${ARROW_SOURCE_HOME}/ci/scripts/r_install_system_dependencies.sh"
 fi
 
-# Install rsync for bundling cpp source and curl to make sure it is installed on all images
-$PACKAGE_MANAGER install -y rsync curl
+# Install rsync for bundling cpp source and curl to make sure it is installed on all images,
+# cmake is now a listed sys req.
+$PACKAGE_MANAGER install -y rsync cmake curl
```

Comment:
If `cmake` isn't new enough on the `test-r-offline-maximal` job, it will still fail. 🤞 this is fine, otherwise we'll want to use install_cmake.sh

Reply:
We should(tm) have new enough cmake the only issue would be centos 7 but as we source devtool set there that should work as well. But we will see^^